yt/yt/core/logging/formatter.cpp

```diff
@@ -205,6 +213,9 @@ i64 TStructuredLogFormatter::WriteFormatted(IOutputStream* stream, const TLogEve
                     .Item("level").Value(FormatEnum(event.Level))
                     .Item("category").Value(event.Category->Name);
             })
+            .DoIf(EnableHostField_, [&] (auto fluent) {
+                fluent.Item("host").Value(NNet::GetLocalHostName());
+            })
```

Comment:
Could you explain the need for this "host" field?
I don't like having it separated from the other system fields. If we want to be able to enable only a subset of system fields, I'd suggest changing the bool option with a flag enum. However, it should be done in a separate PR, this one's already complex enough.

Reply:
It is useful when you have multiple hosts producing logs to the same dynamic table. While this is something that we could append in the specific implementation for dynamic tables, I think it is closer to the other system fields and might be useful in some other log writer later on. 

I don't like it separated either, but I don't want to break existing structured logs. Changing to a flag enum is a good idea, I'll write a TODO.